### PR TITLE
JBrowse screenshots

### PR DESCRIPTION
**Pull request by nathandunn**

### CODE REVIEW

**Comment by nathandunn** [comment]
WIP - for #244

File: src/js_src/containers/genePage/transcriptInlineViewer.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  handleImageLoaded() {
+    this.setState({imageStatus: ''});
+  }
+
+
+  render() {
+    // let externalPrefix = 'http://bw.scottcain.net/jbrowse/?data=data%2F';
+    // let externalPrefix = 'http://34.208.22.23/jbrowse/overview.html?data=data%2F';
+    let externalPrefix = 'http://52.43.125.139/jbrowse/overview.html?data=data%2F';
```
Comment (agr-dev-ops) [inline src/js_src/containers/genePage/transcriptInlineViewer.js]: This hard coded IP should be "jbrowse.alliancegenome.org

File: src/js_src/containers/genePage/transcriptInlineViewer.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    // let internalPrefix = 'http://localhost/jbrowse/overview.html?data=data%2F';
+    // let visualizationPrefix = 'http://dev.alliancegenome.org:8891/?url=';
+    // let visualizationPrefix = 'http://localhost:8891/?url=';
+    let visualizationPrefix = 'http://52.43.125.139:8891/?url=';
```
Comment (agr-dev-ops) [inline src/js_src/containers/genePage/transcriptInlineViewer.js]: Change IP here too

**Comment by nathandunn** [comment]
@a8wright  I made most of the changes and cleaned out some stuff.   You saw my note that: jbrowse.alliancegenome.org doesn't resolve?  

If we can get that in today then great.  Otherwise, we should just use these IP's until next week so that I can demo.

**Pull request merged**